src/Engine/Component/VRDevice.cpp

```diff
@@ -5,7 +5,6 @@
 
 namespace Component {
     VRDevice::VRDevice() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'VRDevice::controllerID' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)